code/processes/segmentedtickerplant.q

```diff
@@ -136,4 +136,5 @@ generateschemas[];
 
 // Create log directory, open all table logs
 // use name of schema to create directory
-.stplg.init[dbname::(string .proc.procname),"_"]
\ No newline at end of file
+`dbname set (string .proc.procname);
```

Comment:
do we need a variable called dbname if it is always just going to be the string of .proc.procname?